Earthfile

```diff
@@ -2,16 +2,16 @@ VERSION 0.7
 
 ARG --global go_version=1.20.4
 ARG --global golangci_lint_version=1.52.2
-ARG --global bufbuild_version=1.17.0
+ARG --global bufbuild_version=1.18.0
 ARG --global migrate_version=4.15.2
-ARG --global sqlfluff_version=2.0.7
+ARG --global sqlfluff_version=2.1.0
 
 FROM golang:$go_version-alpine
 
 deps:
   WORKDIR /translate
   COPY go.mod go.sum .
-  RUN go mod download
+  RUN --mount=type=cache,target=/go/pkg/mod go mod download
```

Comment:
When you use a cache mount, it is constrained to only the RUN task you run. `+go` does not benefit from it - it downloads all dependencies again.

The frequency of changes in go.mod is rare, it's more beneficial to not use a cached mount.